Header header
int16 iFrame
float64  fTimestamp
int16 params
RigidBodyData[] rigid_bodies



